src/requirements.cpp

```diff
@@ -745,17 +745,21 @@ bool requirement_data::has_comps( const read_only_visitable &crafting_inv,
 {
     bool retval = true;
     int total_UPS_charges_used = 0;
-    for( const auto &set_of_tools : vec ) {
```

Comment:
I don't think this file is my changes but it might be anothersimulacrum when they fixed the tests failing for me.